### DISCUSSION TITLE
No consumption data even after many charges

### DISCUSSION THREAD

**Discussion by papakwstas** [teslamate-org/teslamate#2383] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I installed Teslamate and restored data from previous backup and all seem to be OK. Unfortunatelly there is no comsumption data yet although i have completed several AC and DC charges. 
Please see the photos for more. Also  check log for a time discrepancy if relevant.

### Expected Behavior

After many charges completed so far, consuption data should be showing as normal

### Relevant log output

```shell
| 2022-01-27 10:17:14.839 car_id=1 [info] Driving / Ended / 8 km – 46 min
teslamate_1         | 2022-01-27 10:17:14.839 car_id=1 [info] Start / :online
teslamate_1         | 2022-01-27 10:18:40.189 [notice]     :alarm_handler: {:clear, TeslaMate.Vehicles.Vehicle_1_api_error}
teslamate_1         | 2022-01-27 10:20:22.644 car_id=1 [info] Suspending logging
teslamate_1         | 2022-01-27 10:41:13.865 [error] Vehicle Error: Vehicle is offline
teslamate_1         | 2022-01-27 10:41:44.426 [error] Vehicle Error: Vehicle is offline
teslamate_1         | 2022-01-27 10:42:15.047 [error] Vehicle Error: Vehicle is offline
teslamate_1         | 2022-01-27 10:42:45.619 [error] Vehicle Error: Vehicle is offline
teslamate_1         | 2022-01-27 10:43:15.622 car_id=1 [info] Fetching vehicle state ...
teslamate_1         | 2022-01-27 10:43:16.069 car_id=1 [info] Start / :offline
teslamate_1         | 2022-01-27 10:43:16.081 car_id=1 [info] Disconnecting ...
database_1          | 2022-01-27 10:52:40.742 EET [31] LOG:  stats_timestamp 2022-01-27 10:52:40.767749+02 is later than collector's time 2022-01-27 10:52:40.74246+02 for database 0
```

### Screenshots

![charges](https://user-images.githubusercontent.com/58748093/151335414-8f8de8e7-e776-486c-a7bb-00f4ecd5b915.PNG)
![drives](https://user-images.githubusercontent.com/58748093/151335421-c0a41593-7e3a-4997-aaf8-640a763bea29.PNG)
![nodata](https://user-images.githubusercontent.com/58748093/151335424-7dc24cd2-40f8-42b5-a491-bf2e493275d2.PNG)

### Type of installation

Docker

### Version

v.1.25.2

**Comment by adriankumpf** [comment]
Does the Efficiency dashboard show a value in the bottom right corner? If so, consumption data can be calculated. In cold weather (indicated by the blue snowflake), though, consumption values cannot be calculated precisely and might therefore be hidden.

**Reply by papakwstas** [reply]
Please see the pic
![EFFICIENCY](https://user-images.githubusercontent.com/58748093/151352990-2936131a-ae83-4f04-bc3c-1ddf65cdcb11.PNG)

**Reply by adriankumpf** [reply]
In this case, more charges are necessary. Also, our [FAQ](https://docs.teslamate.org/docs/faq/#why-are-no-consumption-values-displayed-in-grafana) was recently updated with this hint which might be relevant:

> Charging sessions have to be longer than 10 minutes and less than 95% state-of-charge (SoC)

**Reply by papakwstas** [reply]
Yes tried that hint and will give a few more tries to see what happens... What changes are you mentioning. Is there something i might do to help out the situation?

**Reply by adriankumpf** [reply]
There simply has to be a certain number of charges whose data yields the same efficiency value. You can try to divide your next charge into small ones (at least 10 minutes long) to speed up the process.

**Reply by cwanja** [reply]
It looks like the primary issue, as Adrian pointed out, is that you are charging up to 100% (or more than 95%) on most of your charges.

**Reply by papakwstas** [reply]
Insisted more and re-charged a couple of times for 20 mins and below 95% as documented and it works. 
Just a question: Since this is an LFP Tesla it should be charged up to 100% each time if possible. Will these charges be omitted by Teslamate or are still taken into account as normal?

**Reply by Bolten88** [reply]
I'm having this issue as well with my LFP tesla Model Y. Should i really charge it less than 100% to get the consumption data?
